playbooks/cloudwatch_server_logs.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Set up aws_logs_agent to send server logs to cloudwatch
+  hosts: all
+  become: true
```

Comment:
Is it possible to use the `become` statement only for tasks that require admin privileges, inside the role? For example, `get_url` might not require them.

Reply:
thanks for raising it, I had followed existing pattern but I agree thatw e should only allow sudo access to required steps. I'll look into it!

Reply:
Done!